proto/translate/v1/translate.proto

```diff
@@ -21,7 +21,13 @@ message Message {
   string id = 1;
   string message = 2;
   string description = 3;
-  bool fuzzy = 4;
+  Status status = 4;
+
+  enum Status {
+    STATUS_UNTRANSLATED = 0;
+    STATUS_FUZZY = 1;
+    STATUS_TRANSLATED = 2;
+  }
```

Comment:
two things:
1. first enum element should be Unspecified
2. be consistent with other the enum (`SCHEMA`), e.g. drop the `STATUS` prefix
```suggestion
  enum Status {
    UNSPECIFIED = 0;
    UNTRANSLATED = 1;
    FUZZY = 2;
    TRANSLATED = 3;
  }
```

Reply:
do we need unspecified, isn't default - untranslated?

Reply:
> do we need unspecified, isn't default - untranslated?

The UNSPECIFIED value is not required, but it is often used as a default value in case the status is not set or is unknown.

> prefixing enum values is preferred based on protobuf style guide.

we with @janishorsts  decided that was too verbose (when I defined `SCHEMA` ), and ditched the prefix. So I think it would be good to remain consistent with that in other enums as well